File: BE_QuanLyTro/src/main/java/ut/edu/be_quanlytro/Dto/Response/ContractCreateResponse.java
package ut.edu.be_quanlytro.Dto.Response;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter @Setter @Builder
public class ContractCreateResponse {
    private UUID contractId;
    private String tenantUsername;
    private String tenantRawPassword;
    private String message;
}

File: Flutter_QuanLyTro/lib/data/models/response/contract_create_response.dart
class ContractCreateResponse {
  final String contractId;
  final String tenantUsername;
  final String tenantRawPassword;
  final String message;

  ContractCreateResponse({
    required this.contractId,
    required this.tenantUsername,
    required this.tenantRawPassword,
    required this.message,
  });

  factory ContractCreateResponse.fromJson(Map<String, dynamic> json) {
    return ContractCreateResponse(
      contractId: json['contractId'] ?? '',
      tenantUsername: json['tenantUsername'] ?? '',
      tenantRawPassword: json['tenantRawPassword'] ?? '',
      message: json['message'] ?? '',
    );
  }
}